Libraries/LPInfra.framework/frameworks-strip.sh

```diff
@@ -2,7 +2,7 @@ APP_PATH="${TARGET_BUILD_DIR}/${WRAPPER_NAME}"
 
 # This script loops through the frameworks embedded in the application and
 # removes unused architectures.
-find "$APP_PATH" -name '*.framework' -type d | while read -r FRAMEWORK
+find "$APP_PATH" -name 'LPAMS.framework' -o -name 'LPInfra.framework' -o -name 'LPMessagingSDK.framework' -type d | while read -r FRAMEWORK
```

Comment:
We are looking into this but why are you not looking to strip LPMonitoring as well?